### ISSUE TITLE
Wrong JDK17 architecture downloaded on M1

### ISSUE THREAD

**Issue by iuliand-db** [VirtusLab/scala-cli#1486]
**Version(s)**
0.1.16

**Describe the bug**
Installed scala-cli via `brew`. It installed an X86 binary, despite using an M1 (ARM) Mac. Then it proceeded to downloading JDK17, despite having JDK 11 available (and that's what I need to use, for work related reasons). Additionally, the JDK17 was also for the wrong (Intel) architecture:

```
Downloading JVM temurin:17
^Ctps://github.com/adoptium/temurin17-binaries/releases/download/jdk-17%2B35/OpenJDK17-jdk_x64_mac_hotspot_17_35.tar.gz
```

**To Reproduce**

See above.

**Expected behaviour**

Use the existing JDK. If downloading something, use the right architecture.

**Comment by Gedochao** [comment]
Hey, the M1 (ARM) Mac `scala-cli` binary isn't available with `brew` just yet. If you want to use it, for now you have to get it from [the relevant release page](https://github.com/VirtusLab/scala-cli/releases/tag/v0.1.16).
Making it available from brew can be tracked in #1430 

Even if you have JDK 11 available on your path, `scala-cli` still needs JDK 17 for `bloop`, so that's expected. the `bloop` fork we use requires Java 17. More on that in #1379 

Now, where you are right is that the downloaded JDK 17 should be an M1-dedicated release, that is definitely a bug, probably related to `coursier`. I'll rename this ticket to reflect that, thanks for pointing that out!

**Comment by lwronski** [comment]
When #1430 is complete, `brew` install ARM native `scala-cli` launcher for M1 and by default Scala CLI `0.1.19` download JDK for architecture on M1.

Feel free to reopen this issue if you still have this issue.
